src/OpenApi/test/Microsoft.AspNetCore.OpenApi.Tests/Services/OpenApiSchemaService/OpenApiSchemaService.ResponseSchemas.cs

```diff
@@ -122,8 +123,8 @@ await VerifyOpenApiDocument(builder, document =>
                 {
                     Assert.Equal("id", property.Key);
                     Assert.Equal(JsonSchemaType.Integer, property.Value.Type);
-                    Assert.Equal(1, property.Value.Minimum);
-                    Assert.Equal(100, property.Value.Maximum);
+                    Assert.Equal("1", property.Value.Minimum);
+                    Assert.Equal("100", property.Value.Maximum);
```

Comment:
Is there a test anywhere that validates these use invariant format for numbers? (e.g. `"1234.56"` not `"1,234.56"` or `"1.234,56"`)

Reply:
Good call -- added some tests elsewhere.